File: fibonacci/fibonacci.php
<?php
	function calc_fibonacci($limit){
		$a = 0;
		$b = 1;
		$t = 0;
		$start_time = hrtime(true);
		for ($i = 0; $i < $limit; $i++){
			$t = $a;
			$a = $b;
			$b = $b + $t;
		}
		$end = hrtime(true);
		$eta = $end - $start_time;
		$eta /= 1e+6;
		error_log("Time: " . $eta);
		return($b);
	}
	$fib = calc_fibonacci(1000);
	error_log("Total Fib: " . $fib);

File: fibonacci/fibonacci.py
import time

def calcFibonacci(limit):
	a = 0;
	b = 1;
	t = 0;
	start_time = time.time()
	for i in range(0, limit):
		t = a;
		a = b;
		b = b + t;
	total_time =  ((time.time() - start_time) * 1000)
	print("Time: ", total_time, ' ms')
	return b

fib = calcFibonacci(1000)
print(fib)